Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py

```diff
@@ -0,0 +1,615 @@
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def disable_user_command(client, args):
+    """
+        Deactivate user by using 'DELETE' to Github API , if Connection to the service is successful.
+
+        Args:   demisto command line argument
+        client: Github API
+
+        Returns:
+            success : success=True, id as iden, active status
+            fail : success=False, id as iden, errorCod, errorMessage, details
+    """
+    scim = verify_and_load_scim_data(args.get('scim'))
+    scim_flat_data = map_scim(scim)
+    user_id = scim_flat_data.get('id')
+
+    if not (user_id):
+        raise Exception('You must provide either the id of the user')
+
+    res = client.disable_user(user_id)
+
+    if res.status_code == 204:
+        generic_iam_context = OutputContext(active=False, success=True, iden=user_id)
+
+    elif res.status_code == 404:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=404,
+                                            errorMessage=USER_NOT_FOUND, details=res.json())
+    else:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=res.status_code,
+                                            errorMessage=res.json().get('detail'), details=res.json())
+
+    generic_iam_context_dt = f'{generic_iam_context.command}(val.id == obj.id && val.instanceName == obj.instanceName)'
+    outputs = {
+        generic_iam_context_dt: generic_iam_context.data
+    }
+
+    readable_output = tableToMarkdown(name='Disable GitHub User:',
+                                      t=generic_iam_context.data,
+                                      headers=["brand", "instanceName", "success", "active", "id",
+                                               "username", "email",
+                                               "errorCode", "errorMessage", "details"],
+                                      removeNull=True)
+
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+
+    params = demisto.params()
+
+    # get the service API url
+    base_url = params.get('url')
+    # get the service token  which is mendatory
+    token = params.get('token')
+    # get the org(organization name)
+    org = params.get('org')
+
+    verify_certificate = not demisto.params().get('insecure', False)
+
+    headers = {
+        'accept': 'application/json',
+        'content-type': 'application/json',
+    }
+
+    proxy = demisto.params().get('proxy', False)
+    command = demisto.command()
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+
+    # Commands supported for GitHub API for user
+    commands = {
+        'test-module': test_module,
+        'get-user': get_user_command,
+        'create-user': create_user_command,
+        'update-user': update_user_command,
+        'enable-user': create_user_command,
```

Comment:
is this the same API call as creating a new user?

Reply:
Yes. there is no disable user for GitHub. GitHub got delete user api which deletes the user from their system. We are renaming it as disable-user to keep it consistent with the other integrations. As there is no enable use in GitHub, we are using enable-user to call create-user to recreate the user in the GitHub system. We are keeping enable-user to make the implementation consistent with other integrations for User provisioning.